pages/products.html

```diff
@@ -0,0 +1,877 @@
+            </li>
+            <li class="pt-2">
+              <input
+                type="radio"
+                id="three-star"
+                name="rating"
+                value="three-star"
+                checked
+              />
+              <label for="three-star">3 stars and above</label>
+            </li>
+            <li class="pt-2">
+              <input
+                type="radio"
+                id="two-star"
+                name="rating"
+                value="two-star"
+                checked
+              />
+              <label for="two-star">2 stars and above</label>
+            </li>
+          </ul>
+
+          <!-- category filter -->
+          <h3 class="headline-sm mt-8">Category</h3>
+          <ul class="d-flex flex-col text-sm">
+            <li class="pt-2">
+              <input type="checkbox" id="monitor" name="monitor" checked />
+              <label for="monitor">Monitor</label>
+            </li>
+            <li class="pt-2">
+              <input type="checkbox" id="chairs" name="chairs" />
+              <label for="chairs">Gaming Chairs</label>
+            </li>
+            <li class="pt-2">
+              <input type="checkbox" id="headphone" name="headphone" />
+              <label for="headphone">Headphones</label>
+            </li>
+            <li class="pt-2">
+              <input type="checkbox" id="keyboard" name="keyboard" />
+              <label for="keyboard">Keyboard</label>
+            </li>
+            <li class="pt-2">
+              <input type="checkbox" id="mouse" name="mouse" />
+              <label for="mouse">Mouse</label>
+            </li>
+          </ul>
+
+          <!-- sort by filter -->
+          <h3 class="headline-sm mt-8">Sort By</h3>
+          <ul class="d-flex flex-col text-sm">
+            <li class="pt-2">
+              <input
+                type="radio"
+                id="high-to-low"
+                name="sort-by"
+                value="high-to-low"
+                checked
+              />
+              <label for="high-to-low">Price High to Low</label>
+            </li>
+            <li class="pt-2">
+              <input
+                type="radio"
+                id="low-to-high"
+                name="sort-by"
+                value="low-to-high"
+                checked
+              />
+              <label for="low-to-high">Price Low to High</label>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <!-- products -->
+      <sction id="products" class="d-grid mt-8">
+        <div class="card-wrapper product-card place-center">
+          <span class="vertical-card-badge text-sm p-3 p-relative"
+            >Best Seller</span
+          >
+          <div class="p-4">
+            <img
+              class="img-responsive d-block m-auto"
+              src="../assets/readgear-headphones.png"
+              alt="headphone"
+            />
+          </div>
```

Comment:
see if you can cut some height here

Reply:
okay. will reduce size of the image and remove extra spacing. thank you for giving suggestion